app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
```

Comment:
Ideally I wanted to make all columns have an automatic width so that all content is flexible and you don't need to put in any constraints. That works the same way a table would work automatically. But as soon as you have a second group of questions, "designers" don't like the questions of both groups not vertically aligning. That's why you would need to give one column a fixed width.